tests/tx/test_indexes.py

```diff
@@ -108,10 +112,11 @@ def test_tx_tips_voided(self):
         tx2.timestamp = int(self.clock.seconds()) + 1
         tx2.resolve()
         self.assertTrue(self.manager.propagate_tx(tx2, False))
-        self.assertEqual(
-            {tx.hash for tx in self.manager.tx_storage.indexes.mempool_tips.iter(self.manager.tx_storage)},
-            {tx2.hash}
-        )
+        if self.manager._enable_sync_v2:
```

Comment:
Isn't it better to check if the index itself exists (instead of sync v2)? It seems safer for future changes.

Reply:
Thanks!